### DISCUSSION TITLE
Suggestion changes + add solar

### DISCUSSION THREAD

**Discussion by Matze5593** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#14] [Ideas]
I have a suggestion for a change. Change radiator to heating circuit01 and add heating circuit02
ich habe einen Änderungsvorschlag. Heizkörper ändern in Heizkreis01 und hinzufügen Heizkreis02

I'm currently working on the solarthermie
an der Solarthermie arbeite ich gerade

Unfortunately you cannot upload files or start a pull request
leider kann man keine Dateien hochladen oder einen pull request starten

![froeling_heizkreis01](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/55891473/e2cf9f93-f734-4f6a-8d3e-99af1f773283)
![froeling_heizkreis02](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/55891473/2781afbf-832d-4224-a96c-12bd39c18018)
[ha_dashboard.yaml.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/13980646/ha_dashboard.yaml.txt)
Heizkreis02

![solarthermie](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/55891473/b304aeac-5be3-441c-948c-653f6d18bd1a)

**Comment by GyroGearl00se** [comment]
Hallo Matze,
Ich habe mir gerade extra einen dummy weiteren GitHub Account erstellt um Änderungen vorzunehmen und einen Pull request zu starten. Funktioniert tadellos :)

Zuerst erstellst du dir einen "Fork". Darin kannst du dann deine Änderungen vornehmen und anschließend eine Pull request erstellen.

Gerne nochmal probieren.

Vorab schonmal vielen Dank für deinen Beitrag.

**Comment by GyroGearl00se** [comment]
Ich habe die Anpassung von Heizkörper > Heizkreis01 durchgeführt. Macht natürlich Sinn um es später besser aufteilen zu können bei mehreren Heizkreisen. - https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/commit/3828684b084ccd85cb2a5961fa18cc0a6b868b8b
...
Ich habe gerade deinen Fork gesehen. Wenn ich die Tage dazu komme werde ich dann auch mal Heizkreis02 & Solarthermie entsprechend hinzufügen.

**Comment by GyroGearl00se** [comment]
Ich habe die Änderungen / Neuerungen von @Matze5593 mit kleinen Modifikationen entsprechend übernommen.

https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/compare/422f8232bea25d59db813284de4c311c816483bf...7411ec103006fdcb071d52548687e6ab37526521



Du hattest den entitätsnamen für froeling_solarthermie.svg als Beschriftung hinzugefügt, jedoch nicht als Kennung (ID). Habe es angepasst und soeben hochgeladen.

https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/commit/dba737a4e664e4e3b046cac428b60bee49559925

![image](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/94406798/ceb17ee0-9327-472f-88bb-774fd967cc87)

Gerne mal testen. (Hinweis: Es kann durchaus vorkommen, dass du die Änderung adhoc nicht siehst wenn du die svg file austauschst. Dies liegt meines Wissens am Browser cache. Demnach eventuell cache leeren, HA neu starten, Browser neu starten)
